lib/commands/slicemachine/commands/ls.js

```diff
@@ -12,26 +12,21 @@ function displayLibs(libs) {
   });
   console.log('');
   consola.info('Something\'s missing?\nCheck paths in your configuration file!\n');
-}
 
+  return libs;
+}
+ 
 /* eslint-disable  consistent-return */
 async function ls() {
-  try {
-    const sm = getSmFile();
-    const expandedLibs = await Promise.all(sm.libraries.map(async (lib) => {
-      try {
-        const slices = await Libraries.slicesModels(lib);
-        const sliceNames = Object.keys(slices);
-        return [lib, sliceNames];
-      } catch (e) {
-        console.error(e);
-      }
-    }));
+  const sm = getSmFile();
+  const extractedLibs = sm.libraries.map((lib) => {
+    const slices = Libraries.slicesModels(lib);
+    const sliceNames = Object.keys(slices);
+
+    return [lib, sliceNames];
+  });
 
-    displayLibs(expandedLibs);
-  } catch (e) {
-    consola.error(e);
-  }
+  return displayLibs(extractedLibs);
```

Comment:
makes no sense, displayLibs returns nothing, it's just a console msg so it won't help.
you need to return extractedLibs eventually if you need to test something here but then it would mean split the logic and the display

Reply:
can do.

Reply:
Done :)